### DISCUSSION TITLE
use-case that requires rebooting of uport ios sdk repository

### DISCUSSION THREAD

**Discussion by sushantgiri** [decentralized-identity/veramo#581] [Q&A]
I am trying to integrate the sdk. I started with the sample project but prompted with error during carthage setup. I am using xcode 12.4. How can I use this sdk. I have implemented the kotlin library but iOS seems to have the problem

**Comment by mirceanis** [comment]
Unfortunately we can no longer provide iOS SDK support. The kotlin sdk is also currently unmaintained.
All our efforts are going into this Veramo framework and we would be happy to help if you want to integrate it into a react-native app.
If you are just starting out, please do not use the iOS and kotlin/android SDKs because there is no support available.
